content/sensu-core/2.0/guides/clustering.md

```diff
@@ -0,0 +1,257 @@
+---
+title: "Clustering"
+linkTitle: "Clustering"
+weight: 120
+version: "2.0"
+product: "Sensu Core"
+platformContent: false
+menu: 
+  sensu-core-2.0:
+    parent: guides
+---
+
+## What is a Sensu Cluster?
+
+A Sensu Cluster is a group of [at least 3][1] sensu-backend nodes, each supporting an embedded or external etcd instance. Creating a Sensu Cluster ultimately configures an [etcd cluster][2].
+
+## Why use clustering?
+
+Clustering is important to make Sensu more highly available, reliable, and durable. It will help you cope with the loss of a backend node, prevent data loss, and distribute the network load of agents.
```

Comment:
Love this section! Although a bit confused around "the network load of agents"

Reply:
Good point. What I'm trying to say is that on a single backend, all agents will consume the same network resources, whereas in a clustered environment, all agents will be distributed across the backends, freeing up some of those network resources. I wanted to avoid using the work "balance" because that might imply we are load balancing agents (which we are not at the moment). I should make a note of that as well!

Reply:
If there's an easier way to say that, I'd love your input 😉

Reply:
I think this was more a gap in my own understanding, so thanks for clarifying! I think we can leave this as-is 👍

Reply:
I did add a note about recommending a load balancer to this section, I think it's relevant here so thanks for pointing that out!